keps/sig-node/606-compute-device-assignment/kep.yaml

```diff
@@ -24,7 +24,7 @@ stage: stable
 # The most recent milestone for which work toward delivery of this KEP has been
 # done. This can be the current (upcoming) milestone, if it is being actively
 # worked on.
-latest-milestone: "v1.20"
```

Comment:
should the stable version be updated 6 lines below?

Reply:
probably yes, let me review. Thanks for pointing this out!